doc/Type/Buf.pod6

```diff
@@ -116,7 +116,7 @@ Defined as:
 
     multi method list(Buf:D:)
 
-Returns a C<Seq> of codepoints.
+Returns a C<List> of codepoints.
```

Comment:
Hmmm...  I wonder why it is a List, and why the documentation mentions codepoints here?  They're just bytes!